tests/tt_eager/python_api_testing/unit_testing/test_bert_ops.py

```diff
@@ -20,304 +20,422 @@
-        (True, True, True, 2688, 1024, 3072, None),
-        (True, True, False, 2688, 1024, 3072, None),
-        (True, False, True, 2688, 1024, 3072, None),
-        (True, False, False, 2688, 1024, 3072, None),
-        # # # # in1-L1-selfout
-        (False, True, True, 2688, 1024, 1024, None),
-        (False, True, False, 2688, 1024, 1024, None),
-        (False, False, True, 2688, 1024, 1024, None),
-        (False, False, False, 2688, 1024, 1024, None),
-        # # # # in1-dram-selfout
-        (True, True, True, 2688, 1024, 1024, None),
-        (True, True, False, 2688, 1024, 1024, None),
-        (True, False, True, 2688, 1024, 1024, None),
-        (True, False, False, 2688, 1024, 1024, None),
-        # # # # in1-L1-ff1
-        (False, True, True, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        (False, True, False, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        (False, False, True, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        (False, False, False, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        # # # # in1-dram-ff1
-        (True, True, True, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        (True, True, False, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        (True, False, True, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        (True, False, False, 2688, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        # # # # # in1-L1-ff1 - no Gelu
-        (False, True, True, 2688, 1024, 4096, None),
-        (False, True, False, 2688, 1024, 4096, None),
-        (False, False, True, 2688, 1024, 4096, None),
-        (False, False, False, 2688, 1024, 4096, None),
-        # # # # in1-dram-ff1 - no Gelu
-        (True, True, True, 2688, 1024, 4096, None),
-        (True, True, False, 2688, 1024, 4096, None),
-        (True, False, True, 2688, 1024, 4096, None),
-        (True, False, False, 2688, 1024, 4096, None),
-        # # # # in1-L1-ff2
-        (False, True, True, 2688, 4096, 1024, None),
-        (False, True, False, 2688, 4096, 1024, None),
-        (False, False, True, 2688, 4096, 1024, None),
-        (False, False, False, 2688, 4096, 1024, None),
-        # # # # in1-dram-ff2
-        (True, True, True, 2688, 4096, 1024, None),
-        (True, True, False, 2688, 4096, 1024, None),
-        (True, False, True, 2688, 4096, 1024, None),
-        (True, False, False, 2688, 4096, 1024, None),
+        (64, 64, 64),
     ],
 )
-@skip_for_wormhole_b0("WH ND hang, see issue #4392")
 def test_bert_linear_batch7(
     device,
-    fidelity,
-    in0_sharded,
-    out_sharded,
-    in1_in_dram,
-    has_bias,
-    fp32_acc_mode,
-    packer_l1_acc,
     M,
     K,
     N,
-    activation,
     function_level_defaults,
 ):
     in0_shape = [1, 1, M, K]
     in1_shape = [1, 1, K, N]
-    bias_shape = [1, 1, N]
-    grid_size = (8, 7)
+    grid_size = (1, 1)
```

Comment:
not using multicore?

Reply:
reverted it back to original code, was a mistake